src/license_text_normalizer/main.py

```diff
@@ -147,6 +151,18 @@ def _strip_words(line: str, words_to_strip: List[str]) -> str:
     return normalized_line.lstrip()
 
 
+def _strip_delimiters_present_anywhere(
+    line: str, delimiters: List[str]
+) -> str:
```

Comment:
I'm concerned that this approach may be too aggressive. Some of the leading delimiters are likely valid parts of a license text if not present at the beginning of a line (consider the semicolon, for example).

I think a safer approach would likely be to do something similar to the existing pattern that strips lines that consist of all one, repeated non-alphanumeric character.

An idea for how to do that could be to have a function that checks if a line consists of all non-alphanumeric characters, no matter if they are repeating or not. And then if it returns `True`, the whole line could be replaced with an empty string. That could likely even replace the existing check for repeating non-alphanumeric characters, because that would then just be a subset of this.